core/backends/backend-nextcloud/src/nextcloudStorage.ts

```diff
@@ -127,6 +93,45 @@ export class NextcloudStorage extends AbstractStorage {
     }
   }
 
+  private async getLastEditDetails(
+    page: string,
+  ): Promise<[Date | undefined, UserDetails | undefined]> {
```

Comment:
I think we should always favor named return values instead of indexed ones
```suggestion
  ): Promise<{date?: Date, userDetails?: UserDetails}> {
```